scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -202,49 +199,231 @@ def setup_single_workspace(workspace):
+                    print(f"Copied {ent_type_before} data table over to: {destination_workspace_namespace}/{destination_workspace_name}")
+                ent_names = []
+            ent_names.append(ent_name)
+            ent_type_before = ent_type
+
+        # copy set Data Table to workspace
+        for etype, enames in set_list.items():
+            fapi.copy_entities(source_workspace_namespace, source_workspace_name, destination_workspace_namespace, destination_workspace_name, etype, enames, link_existing_entities=True)
+            print(f"Copied {etype} data table over to: {destination_workspace_namespace}/{destination_workspace_name}")
+
+        # Check if data tables match
+        new_workspace_response = fapi.get_entities_with_type(destination_workspace_namespace, destination_workspace_name)
+        new_entities = new_workspace_response.json()
+        if entities != new_entities:
+            print(f"Error: Data Tables don't match")
+            error = f"Error: Data Tables don't match"
+
+        # Get original workpace bucket
+        get_bucket_success, get_bucket_message = get_workspace_bucket(source_workspace_name, source_workspace_namespace)
+        original_bucket = json.loads(get_bucket_message)["workspace"]["bucketName"]
+        print(f"Original Bucket: {original_bucket}")
+
+        # Get new workpace bucket
+        new_bucket = migration_data["destination_workspace_bucket"].replace("gs://", "")
+        print(f"New Bucket: {new_bucket}")
+
+        # update bucket links
+        update_entities(destination_workspace_name, destination_workspace_namespace, replace_this=original_bucket, with_this=f"{new_bucket}/{original_bucket}")
+        print("Updated Data Table with new bucket path")
+    except Exception as e:
+        copy_successful = False
+        print(f"Error: {e}")
+        error = f"Error: {e}"
+        original_bucket = "ERROR"
+
+    migration_data_with_dt["source_workspace_bucket"] = original_bucket
+    migration_data_with_dt["copy_data_table"] = copy_successful
+    migration_data_with_dt["data_table_error"] = error
+
+    return migration_data_with_dt
+
+
+def copy_workspaces_workflows(migration_data):
+    """Copy Van Allen Lab workspaces Workflows."""
+    # set workspace and namespace
+    source_workspace_name = migration_data["source_workspace_name"]
+    source_workspace_namespace = migration_data["source_workspace_namespace"]
+    destination_workspace_name = migration_data["destination_workspace_name"]
+    destination_workspace_namespace = migration_data["destination_workspace_namespace"]
+
+    # Set up the migration_data dict
+    migration_data_full = migration_data
+
+    # Default copy_successful
+    copy_successful = True
+
+    # Default Error
+    error = "NA"
+
+    # Get the list of all the workflows
+    try:
+        workflow_list = fapi.list_workspace_configs(source_workspace_namespace, source_workspace_name)
```

Comment:
1. also get list of workflow names from source workspace
```
source_wf_names = []
for item in json.loads(response.text):
      source_wf_names.append(item["methodRepoMethod"]["methodName"])
```
repat after the workflows are copied:
```
dest_wf_list = []
...
```

check if `source_wf_list = dest_wf_list`
if they are the same, return `True, dest_wf_list`, if they are NOT return `False, erro`